packages/playwright-core/src/server/isomorphic/utilityScriptSerializers.ts

```diff
@@ -186,6 +233,10 @@ export function source() {
       return { u: value.toJSON() };
     if (isRegExp(value))
       return { r: { p: value.source, f: value.flags } };
+    for (const [k, ctor] of Object.entries(typedArrayConstructors) as [TypedArrayKind, Function][]) {
```

Comment:
I'd start with looking up constructor.name in a set instead of iterating

Reply:
that won't work on Firefox, see impl of `isTypedArray`

Reply:
It does work for me, are you saying this is also due to utility worlds? Something is fishy here. Do we have a bug in the way we initialize utility context? It needs to be proper JS context.

Reply:
as discussed in the meeting, firefox is just off here. let's not fix in this PR.